jvm.c

```diff
@@ -266,8 +266,19 @@ stack_entry_t *execute(method_t *method,
             class_name = find_method_info_from_index(index, clazz, &method_name,
                                                      &method_descriptor);
 
-            class_file_t *target_class =
-                find_or_add_class_to_heap(class_name, prefix);
+            class_file_t *target_class;
+            if (find_or_add_class_to_heap(class_name, prefix, &target_class)) {
+                /* Call static initialization */
+                method_t *method = find_method("<clinit>", "()V", target_class);
+                if (method) {
```

Comment:
Rewrite as following:
```c
if (!method)
    contnue;

local_variable_t own_locals[method->code.max_locals];
...
```

It can reduce the indention.

Reply:
Sorry, I can't understand. How can I use `continue` in `if` block, and only skip the code in `if` and run remaining code in the `case`.

Reply:
It was my bad. I thought there was an iteration.